lib/trace_events.js

```diff
@@ -70,8 +71,7 @@ class Tracing {
 }
 
 function createTracing(options) {
-  if (typeof options !== 'object' || options == null)
```

Comment:
This changes the validation. Before `undefined` was an error, now it would pass.

Reply:
By the way, do we need to unify the error message ? `object` or `Object` :)